docs/sql/index.md

```diff
@@ -1,407 +0,0 @@
-# SQL Support
```

Comment:
Note, I deleted this file and replaced it with a new one for an easier to review diff. I'm not sure if this will render. I will rename back to `index.md` before merging.